src/stactools/sentinel1/rtc_metadata.py

```diff
@@ -135,9 +136,10 @@ def platform(self) -> Optional[str]:
         platformMap = dict(S1A='sentinel-1a', S1B='sentinel-1b')
         return platformMap[self.metadata['MISSION_ID']]
 
-    @property
-    def proj_bbox(self) -> Optional[str]:
-        return self.metadata['PROJ_BBOX']
+    # NOTE: redundent with projection.transform
+    # @property
+    # def proj_bbox(self) -> Optional[str]:
+    #    return self.metadata['PROJ_BBOX']
```

Comment:
Since it's redundant, can we remove it (unless there's some reason to keep it in but commented)? Same w/ https://github.com/stactools-packages/sentinel1/pull/2/files#diff-68f6a98ce4447e3244540fbae6c5c887bde637ea0e4355be054f44bf8313aca4R33.